.testcoverage.yml

```diff
@@ -23,5 +23,6 @@ exclude:
     - tests/internal/testenvironment/
     - tests/internal/testopeninference/openai_proxy.go
     - internal/testing/
+    - tests/internal/testupstreamlib/
```

Comment:
do we want this?

Reply:
can you not add to this unless it's difficult? I don't think it's difficult to cover the lines you added